eng/Packages.Data.props

```diff
@@ -163,6 +163,7 @@
     <!-- TODO: Make sure this package is arch-board approved -->
     <PackageReference Update="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.35.0" />
     <PackageReference Update="Microsoft.IdentityModel.Tokens" Version="6.35.0" />
+    <PackageReference Update="OpenAI" Version="2.0.0-beta.7" />
```

Comment:
Please let me know if there is a better place to put this PackageReference.

Reply:
Do you know offhand how this will affect the AOAI dependency on SCM?  Will it always get the version OAI is referencing and not the repo-wide version?

Reply:
The Azure OpenAI libray takes a project dependency on the OpenAI Nuget package. This means it will pull in whatever version of SCM the OpenAI Nuget package depends on which is currently 1.1.0-beta.4

Reply:
It would be good to understand if that will have any effect on our CI pipelines which run against both released versions and source -- will they catch build breaks introduced by changes to SCM code?

We have some circular dependency issues with the autorest.csharp repo that I'm wondering if they would show up here where libraries depend on one version for source and another for package -- but maybe those issues only apply where there is shared source -- I'm not able to think offhand of a scenario where a unidirectional package dependency could get us into a problem -- unless the CI's are pointing to different versions.

Reply:
> Do you know offhand how this will affect the AOAI dependency on SCM? Will it always get the version OAI is referencing and not the repo-wide version?

I'm actually not sure if our CI pipeline would do anything to change the version of SCM that is being used. Do you know who could help us clarify this?